devops/azure-pipelines/build-stage-template.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+- stage: Build
+  displayName: ${{ parameters.displayName }}
+  jobs:
+  - job: Build
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '10.x'
+      displayName: 'Install Node.js'
+
+    - script: |
+        npm install
```

Comment:
since there's a [package lock](https://docs.npmjs.com/cli/v6/configuring-npm/package-locks) file, [`npm ci`](https://docs.npmjs.com/cli/v6/commands/npm-ci) is preferable to `npm install` so the build is more consistent.

i don't expect you to just know this, but take 5-10 minutes with those links to understand roughly what's going on.
the package.json file declares the direct dependencies of this app with ranges of acceptable versions. devs edit this by hand or via the npm cli. the package-lock file declares what exact versions were discovered and used for all direct and nested dependencies the last time someone ran npm install and committed the changes it made to the lock file. that file is entirely computer-generated.